lisa/schema.py

```diff
@@ -831,6 +831,10 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         default_factory=partial(search_space.IntRange, min=0),
         metadata=field_metadata(decoder=search_space.decode_count_space),
     )
+    zones: search_space.SetSpace[int] = field(
```

Comment:
The zones should be in AzureNodeSchema, because it's not like a general concept.

Reply:
If I move zones then how can I set the VM capability in `_resource_sku_to_capability` since that returns a `NodeSpace`

Reply:
Try `node_space.get_extended_runbook(AzureNodeSchema, type_name=AZURE)` to get extended node space.